src/desktop/apps/artwork/components/commercial/index.styl

```diff
@@ -71,20 +71,20 @@
       padding distance(gutter / 2) 0
   
       > a
-        fine-faux-underline()
         color gray-darker-color
         font-size 15px
-
+        text-decoration underline
```

Comment:
Are these intentional? Seems like we lost the faux underline (which let content 'breathe') a bit, and replaced it with regular `text-decoration underline`.

It looks a bit cramped now, we were discussing in #front-end.